htdocs/web_portal/views/downtime/add_downtime.php

```diff
@@ -67,7 +67,13 @@
             &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<mark><label id="schedulingStatusLabel"></label></mark>
         </div>
 
-
+        <div class="input-warning" id="invalidSelection">
+            <p style="color: #D31F1F;">
+                WARNING: You are NOT allowed to create downtimes
+                when you have selected multiple sites
+                using the <q>Site Timezone</q> option.
```

Comment:
```suggestion
                When selecting multiple sites, you must use the
                <q>Enter Times In: UTC<q> option.
```

Reply:
Changed the warning message in 484394936858aec44dacc18e35860bfe5998b29e